app/server/models/user.py

```diff
@@ -99,9 +99,13 @@ class User(ManyOrgBase, ModelBase):
         secondary=user_transfer_account_association_table,
         back_populates="users")
 
-    # TODO: work out if this should be deprecated
     default_transfer_account_id = db.Column(db.Integer, db.ForeignKey('transfer_account.id'))
 
+    default_transfer_account = db.relationship('TransferAccount',
```

Comment:
@enjeyw looks like we can make the primaryjoin a string as well, so that avoids the dependency issue. and then we're able to do the line you suggested yayy